packages/sdk/src/Loyalties.ts

```diff
@@ -1,38 +1,49 @@
+import * as T from './types/Loyalties'
+
 import { encode, omit } from './helpers'
 import type { RequestController } from './RequestController'
 
 export class Loyalties {
 	constructor(private client: RequestController) {}
 
-	public list(params: $FixMe) {
-		return this.client.get('/loyalties', params)
+	public list(params: T.LoyaltiesListParams) {
```

Comment:
It should not be required to pass this `params` since in old SDK it was not required.